azkaban-common/src/main/java/azkaban/executor/ExecutionReference.java

```diff
@@ -18,16 +18,22 @@
 
 public class ExecutionReference {
```

Comment:
We want to avoid interface changes in ExecutorManager, due to which we need to keep this class.